.github/workflows/ci.yaml

```diff
@@ -12,7 +12,7 @@ jobs:
     steps:
       - uses: actions/setup-go@v1
         with:
-          go-version: '1.13'
+          go-version: '1.14'
```

Comment:
There's another one of these inside test-windows.

Reply:
Thanks.